fdbserver/include/fdbserver/IPager.h

```diff
@@ -17,20 +17,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#pragma once
```

Comment:
move it after #ifdef ?

Reply:
Either one of `#pragma once` and `#ifdef` avoid the header being include multiple times, and `#pragma once` is the more modern one, but maybe none standard. As `#pragma once` look cleaner, I think it make sense to prefer using it than `#ifdef`. That's why I have it go first. Any particular reason you think the two directive should change place?

Reply:
I remember this discussion happened in past and am forgetting the details, but #pragma doesn't work well with some compilation flag etc. Hence, both are added in all other files, I called out to make sure this file maintain the code uniformity :)